CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing
+
+This project adheres partly to the [Gitflow Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows#gitflow-workflow)
+
+## Adding a new feature or bug fix
+
+##### 1. Check out the latest `dev` branch
+
+##### 2. Create a new feature branch
+- Give a short but descriptive name relative to the feature you are working on
+- Append the issue number relating to the feature/fix
+- EG: `delete-user-#125`
```

Comment:
Branches have to follow the convention of `<issue_number>-<description>` or `<description>-#<issue_number>` to be automated by Waffle.io